src/compass/s1_cslc_qa.py

```diff
@@ -0,0 +1,177 @@
+                       'azimuth_fm_rate_mismatch']
+
+        # compute stats and write to hdf5
+        self.stats_dict['corrections'] = {}
+        corrections_dict = self.stats_dict['corrections']
+        for correction in corrections:
+            # create dict for current correction
+            corrections_dict[correction] = {}
+            correction_dict = corrections_dict[correction]
+
+            # get dataset and compute stats according to dtype
+            src_path = f'{ROOT_PATH}/corrections/{correction}'
+            ds = cslc_h5py_root[src_path]
+            stat_obj = isce3.math.StatsFloat32(ds[()].astype(np.float32))
+
+            # write stats to HDF5
+            vals = [stat_obj.mean, stat_obj.min, stat_obj.max]
+            for val_name, val in zip(self.stat_names, vals):
+                correction_dict[val_name] = value_description_dict(
+                    val, f'{val_name} of {correction} correction')
+
+
+    def raster_pixel_classification(self):
+        '''
+        Place holder for populating classification of geocoded pixel types
+        '''
+        self.classification_count_dict['topo'] = {}
+        topo_dict = self.classification_count_dict['topo']
+        topo_dict['percent_layover_pixels'] = value_description_dict(
+            0, 'Percentage of output pixels labeled layover')
+        topo_dict['percent_shadow_pixels'] = value_description_dict(
+            0, 'Percentage of output pixels labeled shadow')
+        topo_dict['percent_combined_pixels'] = value_description_dict(
+            0, 'Percentage of output pixels labeled layover and shadow')
+        self.classification_count_dict['percent_land_pixels'] = \
+            value_description_dict(
+                0, 'Percentage of output pixels labeld as land')
+        self.classification_count_dict['percent_valid_pixels'] = \
+            value_description_dict(
+                0, 'Percentage of output pixels are valid')
+
+
+    def populate_rfi_dict(self):
+        '''
+        Place holder for populating SAFE RFI information
+        '''
+        self.rfi_dict['is_rfi_info_available'] = value_description_dict(
+            True, 'Whether or not RFI information is available')
+        # Follow key/values only assigned if RFI info is avaiable
+        self.rfi_dict['rfi_mitigation_performed'] = value_description_dict(
+            True, 'Whether or not the RFI mitigation step was performed')
+        self.rfi_dict['rfi_mitigation_domain'] = value_description_dict(
+            True, 'Domain the RFI mitigation step was performed')
+        self.rfi_dict['rfi_burst_report'] = value_description_dict(
+            '', 'Burst RFI report')
+
+
+
+    def set_orbit_type(self, cfg):
+        '''
+        Populate QA orbit information
+        '''
+        orbit_file_path = cfg.orbit_path[0]
+        if 'RESORB' in Path(orbit_file_path).name:
+            orbit_type = 'restituted orbit file'
+        if 'POEORB' in orbit_file_path:
```

Comment:
ah shoot, sorry I missed adding this suggestion too: 

```suggestion
        if 'POEORB' in Path(orbit_file_path).name:
```